spec.html

```diff
@@ -7315,8 +7315,7 @@ <h1>HostCallJobCallback ( _jobCallback_, _V_, _argumentsList_ )</h1>
 
     <emu-clause id="sec-hostenqueuepromisejob" aoid="HostEnqueuePromiseJob">
       <h1>HostEnqueuePromiseJob ( _job_, _realm_ )</h1>
-      <p>HostEnqueuePromiseJob is a host-defined abstract operation that schedules the Job Abstract Closure _job_ to be performed, at some future time. The Abstract Closures used with this algorithm are intended to be related to the handling of Promises, or otherwise, to be scheduled with equal priority to Promise handling operations.</p>
-      <p>The _realm_ parameter is either *null* or a Realm Record.</p>
+      <p>The host-defined abstract operation HostEnqueuePromiseJob takes arguments _job_ (a Job Abstract Closure) and _realm_ (a Realm Record or *null*). It schedules _job_ to be performed, at some future time. The Abstract Closures used with this algorithm are intended to be related to the handling of Promises, or otherwise, to be scheduled with equal priority to Promise handling operations.</p>
```

Comment:
```suggestion
      <p>The host-defined abstract operation HostEnqueuePromiseJob takes arguments _job_ (a Job Abstract Closure) and _realm_ (a Realm Record or *null*). It schedules _job_ to be performed at some future time. The Abstract Closures used with this algorithm are intended to be related to the handling of Promises, or otherwise, to be scheduled with equal priority to Promise handling operations.</p>
```